src/Relation/Binary/Construct/Interior/Symmetric.agda

```diff
@@ -0,0 +1,91 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Symmetric interior of a binary relation
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Binary.Construct.Interior.Symmetric where
+
+open import Function.Base using (flip)
+open import Level
+open import Relation.Binary
+
+private
+  variable
+    a b c ℓ r s t : Level
+    A : Set a
+    R : Rel A r
+    S : Rel A s
+    T : Rel A t
+
+------------------------------------------------------------------------
+-- Definition
+
+record SymInterior (R : Rel A ℓ) (x y : A) : Set ℓ where
+  constructor _,_
+  field
+    lhs≤rhs : R x y
+    rhs≤lhs : R y x
+open SymInterior public
+
+------------------------------------------------------------------------
+-- Properties
+
+-- The symmetric interior is symmetric.
+symmetric : Symmetric (SymInterior R)
+symmetric (r , r′) = r′ , r
+
+-- The symmetric interior of R is greater than (or equal to) any other symmetric
+-- relation contained by R.
+unfold : Symmetric S → S ⇒ R → S ⇒ SymInterior R
+unfold sym f s = f s , f (sym s)
```

Comment:
`unfold` is usually reserved for a definition/function/proof which witnesses the equational unfolding/rewrite property of a... function

.. as such, it seems sub-optimal here.

Why not `interior-greatest` or `sym$\subseteq$interior` or even `symmetric⇒interior`...?

Reply:
Or even `interior⁺` (it's a 'constructor'/introduction rule along the axis of relational implication...)?

Reply:
The name is inspired by [`fold` in `Relation.Binary.Construct.Closure.Symmetric`](http://agda.github.io/agda-stdlib/Relation.Binary.Construct.Closure.Symmetric.html#1365). Better name vs consistency.

Reply:
Ah yes, an inductive/least  notion yields`fold` (rather than `elim`) while a conductive/greatest yields `unfold` (rather than `intro` etc...)...

... another round in the battle for hearts and minds between an Oxford/haskell lexicon, and a Stockholm/Göteborg/MLTT one...